RuntimeException in ActiveRecord constructor

There is an extra dirty flag in the array. 

03-20 11:02:05.219: E/AndroidRuntime(2708): Caused by: java.lang.RuntimeException: bad array lengths
03-20 11:02:05.219: E/AndroidRuntime(2708):     at android.os.Parcel.readBooleanArray(Parcel.java:614)
03-20 11:02:05.219: E/AndroidRuntime(2708):     at com.justeat.app.data.BasketsRecord.<init>(Unknown Source)

--- a/plugins/com.robotoworks.mechanoid.db/src/com/robotoworks/mechanoid/db/generator/ActiveRecordGenerator.xtend
+++ b/plugins/com.robotoworks.mechanoid.db/src/com/robotoworks/mechanoid/db/generator/ActiveRecordGenerator.xtend
@@ -264,7 +264,7 @@ class ActiveRecordGenerator {
 				«ENDIF»
 				«ENDFOR»
 				
-				boolean[] dirtyFlags = new boolean[«stmt.columnDefs.size»];
+				boolean[] dirtyFlags = new boolean[«stmt.columnDefs.size - 1»];
 				in.readBooleanArray(dirtyFlags);
 				«FOR col : stmt.columnDefs.filter([!it.name.equals("_id")])»
 				m«col.name.pascalize»Dirty = dirtyFlags[«counter = counter + 1»];

